tests/test_C2_economic_functions.py

```diff
@@ -115,6 +136,41 @@ def test_annuity():
     assert A == present_value * crf
 
 
+def test_get_replacement_costs_equal_lifetimes():
+    replacement_costs = C2.get_replacement_costs(
+        age_of_asset=0,
+        project_lifetime=10,
+        asset_lifetime=10,
+        first_time_investment=100,
+        discount_factor=1,
+    )
+    assert replacement_costs == 0
+
+
+def test_get_replacement_costs_one_reinvestment():
+    replacement_costs = C2.get_replacement_costs(
+        age_of_asset=5,
+        project_lifetime=10,
+        asset_lifetime=10,
+        first_time_investment=100,
+        discount_factor=0,
+    )
+    exp = 100 - 100 / 10 * 5
```

Comment:
It is not clear here if 10 is project_lifetime or asset_lifetime

Reply:
Added comment in e48e05ab50c5b60694295cab4f8bbbcc8c6b8e1c, PR #525